### PR TITLE
Add a README for orchestrator to help people understand it

### PR DESCRIPTION
**Pull request by joelverhagen**
I was very inspired by @dannyjdev's README he wrote for his internal job.

The goal is not perfection. The goal is to simply improve the current situation.

### CODE REVIEW

File: src/NuGet.Services.Validation.Orchestrator/README.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+   1. If any failed, stop doing any further validations.
+   1. If any succeeded, start non-started validations that now have their dependency validations completed.
+1. Start any non-started validations that have all dependency validations completed.
+1. If any validations are still in-progress, re-enqueue and schedule a new Service Bus message so that
+   the validation set can be checked again soon.
+1. If a validation failed, mark the package as `FailedValidation` and send an email to the owner(s).
+1. If all validations succeeded, mark the package as `Available` and send an email to the owner(s).
+   1. Note that part of this successful end result is updating the package's `LastEdited` time, which triggers
+      the [Db2Catalog job](https://github.com/NuGet/NuGet.Services.Metadata/blob/master/src/Ng/Jobs/Db2CatalogJob.cs)
+	  to begin processing it in V3.
```
Comment (loic-sharma) [inline src/NuGet.Services.Validation.Orchestrator/README.md:59]: Is the tab here intentional?

File: src/NuGet.Services.Validation.Orchestrator/README.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+## List of Validators
+
+The easiest way to find validators is to find types implementing `IValidator` or `IProcessor` but here is the current
```
Comment (loic-sharma) [inline src/NuGet.Services.Validation.Orchestrator/README.md:63]: The "current" wording may be confusing in the future. Maybe:

```suggestion
The easiest way to find validators is to find types implementing `IValidator` or `IProcessor` but here is the list as of April 17th, 2020.
```

File: src/NuGet.Services.Validation.Orchestrator/README.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+list.
```
Comment (loic-sharma) [inline src/NuGet.Services.Validation.Orchestrator/README.md:64]: The "current" wording may be confusing in the future. Maybe:

```suggestion
The easiest way to find validators is to find types implementing `IValidator` or `IProcessor` but here is the list as of April 17th, 2020.
```

```suggestion
The easiest way to find validators is to find types implementing `IValidator` or `IProcessor` but here is the list
as of April 17th, 2020.
```

File: src/NuGet.Services.Validation.Orchestrator/README.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+   the validation set can be checked again soon.
+1. If a validation failed, mark the package as `FailedValidation` and send an email to the owner(s).
+1. If all validations succeeded, mark the package as `Available` and send an email to the owner(s).
+   1. Note that part of this successful end result is updating the package's `LastEdited` time, which triggers
+      the [Db2Catalog job](https://github.com/NuGet/NuGet.Services.Metadata/blob/master/src/Ng/Jobs/Db2CatalogJob.cs)
+      to begin processing it in V3.
+
+## List of Validators
+
+The easiest way to find validators is to find types implementing `IValidator` or `IProcessor` but here is the as
```
Comment (agr) [inline src/NuGet.Services.Validation.Orchestrator/README.md:63]: Missing some word at the end of the line?

**Comment by joelverhagen** [comment]
> I would replace the word 'virus' everywhere with 'malware' as a more generic term.

Good idea, done.

> Also, maybe add the [command line arguments](https://github.com/NuGet/NuGet.Jobs/commit/c5251db523bf3d6e1c40e7167ea1655c1811d76d) description? Ideally, we need configuration file covered as well, but that would be another big effort.

Wow, awesome. Copied in.

**Pull request merged**